### DISCUSSION TITLE
PIM vs Entitlement packages

### DISCUSSION THREAD

**Discussion by krukowskid** [AzureCommunityPL/group#297] [Pomoc] [answered]
Cześć,
ze względu na konieczność wdrożenia PIM, zacząłem zagłębiać się w inne usługi dostępne w AAD P2 i trafiłem na entitlement packages. Po przeklikaniu entitlement packages wydaje mi się że jest to bardziej przyjazne dla użytkownika końcowego, a nawet od strony zarządzania przyjemniej się to konfiguruje. Czy ktoś mógłby powiedzieć dwa słowa jak w praktyce wygląda używanie PIM vs entitlement packages? W jakich przypadkach sprawdza się dana usługa? Z początku myślałem, żeby oprzeć wszystko o PIM - eigible na grupy dające wyższe dostępy np. jak DEV musi zweryfikować app insights z prod itd, a active na grupy, które dają dostęp do projektów itd, ale wychodzi na to, że chyba lepiej to rozdzielić.

**Comment by przybylskirobert** [comment, marked as answer]
Hej, jesli temat jeszcze aktualny to  wydaje mi sie ze chodzi ci o PIM oraz Entitlement Managment. 

- Jesli chodzi o PIM to sprawa prosta mozesz zarzadzic rolami w Azure AD oraz na resoucach, oraz 'PAG' ktore sa w preview.
- Jesli chodzi o ELM (entitlement management) tutaj masz duzo opcji 

ELM sklada sie z kliku funkcjonalnosci:
- Connected Organisations - gdzie mozesz targetowac paczki na okreslone tenanty, 
- Catalogs - gdzie mozesz wybrac zasoby z twojego srodowiska ( grupy , site SPO, applikacje) i przypisac do okreslonego katalogu, 
- Access packages - miejsce gdzie robisz packzi ktore sa dostepne da Internal/Extenral users i masz mozliwosc targetowania po userach grupach organizacjach. 

Paczek mozesz uzywac do np:
1. Wspolpraca z kontami z poza twojej firmy - mozesz zarzadzic np ze osoby z tenantu A moga miec dostep do okreslonych paczek i je requestowac, dodatkowo mozesz miec fajnie zarzadzone usuwania konta goscia z twojego tenanta po wygasnieciu paczki. 
2. Uzycie Access packages wewnetrznie na zasadzie chcesz dostep do zarequestuj - jak ktos da ci approval to dostaniesz jak nie to nie :)

Generalnie cale Identity Governance jest fajnym temamtem który opisalem na blogu: 
https://www.azureblog.pl/tag/identity-governance/

PS jest do tego beta API i da sie to oprogramowac
